projects/plugins/jetpack/extensions/blocks/mailchimp/block.json

```diff
@@ -0,0 +1,59 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "jetpack/mailchimp",
+	"title": "Mailchimp",
+	"description": "A form enabling readers to join a Mailchimp audience.",
```

Comment:
Still on the topic of phrasing, here I think we could say "Add a form" to be consistent with other blocks. Again though, that's probably something we should do in a follow-up PR.